peripherals/syscfg/syscfg_l0x0.yaml

```diff
@@ -0,0 +1,154 @@
+# System configuration controller and COMP registers for L0 family
+SYSCFG:
+  CFGR1:
+    BOOT_MODE:
+      MainFlash: [0, "Main Flash memory boot mode"]
+      SystemFlash: [1, "System Flash memory boot mode"]
+      SRAM: [3, "Embedded SRAM boot mode"]
+    MEM_MODE:
+      MainFlash: [0, "Main Flash memory mapped at 0x0000_0000"]
+      SystemFlash: [1, "System Flash memory mapped at 0x0000_0000"]
+      SRAM: [3, "Embedded SRAM mapped at 0x0000_0000"]
+  CFGR2:
```

Comment:
It looks like the L0x0 doesn't have any of these I2C FMP bits, and instead does have FWDIS (though it's incorrectly named FWDISEN in the SVD), so these definitions for I2C FMP should be removed and replaced with FWDIS (though it looks like we'll also need a fix for FWDISEN->FWDIS, and also remove these bits and the CAPA bits from the SVD).

Reply:
Okay, unsure if I parsed this correctly; so here's what I did:
I remove the FMP bits
I renamed FWDISEN to FWDIS
I added a _delete for CAPA.